src/app/work-item/work-item-quick-add/work-item-quick-add.component.ts

```diff
@@ -1,21 +1,22 @@
-import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
-import { ActivatedRoute, Params } from '@angular/router';
+import { Component, EventEmitter, Input, OnInit, Output } from "@angular/core";
+import { ActivatedRoute, Params } from "@angular/router";
 
-import { Logger } from '../../shared/logger.service';
-import { WorkItem } from '../work-item';
-import { WorkItemService } from '../work-item.service';
+import { Logger } from "../../shared/logger.service";
+import { WorkItem } from "../work-item";
+import { WorkItemService } from "../work-item.service";
 
 @Component({
-  selector: 'work-item-quick-add',
-  templateUrl: '/work-item-quick-add.component.html',
-  styleUrls: ['/work-item-quick-add.component.scss']
+  selector: "work-item-quick-add",
+  templateUrl: "./work-item-quick-add.component.html",
+  styleUrls: ["./work-item-quick-add.component.scss"]
 })
 export class WorkItemQuickAddComponent implements OnInit {
-  @Input() workItem: WorkItem;
+  //@Input() workItem: WorkItem;
```

Comment:
It is is failing the tests, then generally you can fix it. However this PR is for setup of initial test framework. So it should not change code unless it is necessary.